extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -52,26 +65,48 @@ class AmpJWPlayer extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    const el = this.element;
     this.contentid_ = userAssert(
-        (this.element.getAttribute('data-playlist-id') ||
-      this.element.getAttribute('data-media-id')),
+        (el.getAttribute('data-playlist-id') ||
+        el.getAttribute('data-media-id')),
         'Either the data-media-id or the data-playlist-id ' +
-      'attributes must be specified for <amp-jwplayer> %s',
-        this.element);
+        'attributes must be specified for <amp-jwplayer> %s',
+        el);
 
     this.playerid_ = userAssert(
-        this.element.getAttribute('data-player-id'),
+        el.getAttribute('data-player-id'),
         'The data-player-id attribute is required for <amp-jwplayer> %s',
-        this.element);
+        el);
+
+    this.contentSearch_ = el.getAttribute('data-content-search') ||
+        '';
+    this.contentContextual_ = el.getAttribute('data-content-contextual') ||
+        false;
+    this.contentRecency_ = el.getAttribute('data-content-recency') ||
+        '';
+    this.contentBackfill_ = el.getAttribute('data-content-backfill') ||
+        false;
```

Comment:
New attributes should be documented in amp-jwplayer.md

Reply:
@robwalch Will add.

Reply:
And also, the example page, referenced in the ticket, will need a PR as well.